README.md

```diff
@@ -30,6 +30,13 @@ find . -name "*_shpec.sh" | entr shpec
 [shpec's own tests](https://github.com/rylnd/shpec/tree/master/shpec/shpec_shpec.sh)
 are a great place to start. More detailed examples to come.
 
+### Mocking
```

Comment:
I like trello, but we're already using Issues to track features and bugs. Is there a specific reason for wanting a trello board?